source/adios2/engine/hdf5/HDF5WriterP.h

```diff
@@ -86,22 +85,30 @@ class HDF5Writer : public Engine
     void Write(const std::string variableName,
                const std::complex<long double> *values);
 
+    void Advance(float timeout_sec = 0.0);
```

Comment:
See https://github.com/ornladios/ADIOS2/wiki/Variables-Scope,-Functions,-and-Namespaces for function parameters using `camelCase`